### PR TITLE
Add new option to unset node groups as managed fields for EKS Cluster

### PR DESCRIPTION
**Pull request by mjura**
Add new option to unset node groups as managed fields for imported EKS Cluster

Issue: https://github.com/rancher/eks-operator/issues/469

### CODE REVIEW

**Review by salasberryfin — APPROVED** [review]

File: eks-upgrade-using-kubectl/README.md [hunk 1]
```diff
@@ -29,13 +31,20 @@ export RANCHER_KUBE="<PATH TO YOUR RANCHER KUBECONFIG>"
...
-./eks-support.sh upgrade -k $RANCHER_KUBE --from 1.22 --to 1.23 --name richtest1
+./eks-support.sh upgrade -k $RANCHER_KUBE --from 1.22 --to 1.23 --nname <EKS_CLUSTER_NAME>
 # For v1
-./eks-support.sh upgrade -k $RANCHER_KUBE --from 1.22 --to 1.23 --name richtest1 --kev1
+./eks-support.sh upgrade -k $RANCHER_KUBE --from 1.22 --to 1.23 --name <EKS_CLUSTER_NAME> --kev1
 ```
 
 > Replace the values of --from, --to and --name with your values.
+
+### Unsetting Node Groups as menaged fields for imported EKS Clusters (only for KEv2)
```
Comment (richardcase) [inline eks-upgrade-using-kubectl/README.md:45]:
```suggestion
### Unsetting Node Groups as managed fields for imported EKS Clusters (only for KEv2)
```

File: eks-upgrade-using-api/README.md [hunk 2]
```diff
@@ -41,19 +41,29 @@ export RANCHER_API="<YOUR RANCHER API ENDPOINT>"
...
 > Replace the values of --from, --to and --name with your values.
 
-6. The cluster will start to upgrade. You can check the status of a specific cluster using this command:
+2. The cluster will start to upgrade. You can check the status of a specific cluster using this command:
 
 ```bash
 ./eks-support.sh status -t $RANCHER_TOKEN --endpoint $RANCHER_API --name richtest1
 ```
+
+### Unsetting Node Groups as menaged fields for imported EKS Clusters (only for KEv2)
```
Comment (richardcase) [inline eks-upgrade-using-api/README.md:63]:
```suggestion
### Unsetting Node Groups as managed fields for imported EKS Clusters (only for KEv2)
```

File: eks-upgrade-using-kubectl/README.md (hunk 1 above)
Comment (mjura) [reply eks-upgrade-using-kubectl/README.md]: fixed

File: eks-upgrade-using-api/README.md (hunk 2 above)
Comment (mjura) [reply eks-upgrade-using-api/README.md]: fixed

**Review by dkeightley — APPROVED** [review]

**Pull request merged**